docs/developer_guide/ConfiguringPerceptionTopics.md

```diff
@@ -128,6 +129,39 @@ not supported.
 `"0.0"`, meaning no randomization.
 
 **[Example](https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/scenario_test_runner/scenario/Property.detectedObjectPositionStandardDeviation.yaml)** -
+
+## Property `*PublishingDelay`
+**Summary** - Delays the publication of the perception topic by the specified
+number of seconds. There are two perception topic delay properties.
+
+- `detectedObjectPublishingDelay` delays the publication of detected object.
+- `pointcloudPublishingDelay` delays the publication of no_ground pointcloud.
+
+**Purpose** - In a real vehicle, the perception topic is generated through the
+processing of a sensing and perception module. However, in `scenario_simulator_v2`,
+these processes are simplified. Detected objects are directly outputted as they
+are from the simulator's objects, and no_ground pointcloud is the result of a
+simple ray tracing. While this is suitable for testing planning and control modules
+in an idealized environment, it does not account for the processing delays in
+perception in a real vehicle. These properties can introduce an arbitrary delay
+in the perception topic to simulate these real-world delays.
+
+**Specification** - For each frame, if the current stamp exceeds the set time
+from the topic's stamp, the topic is published, thereby implementing the delay.
+This could result in a maximum error of 2/FPS. Also, the order of topics to be
+published is guaranteed to be the same as when there is no delay.
+Since the delay is set to the same value for each object, it is not possible to
+delay only a specific object.
+
+**Note** - This feature sets the delay from when the perception results are
+obtained in `scenario_simulator_v2` to when they are published. Not that there
+are other potential sources of delay, such as frame rates becoming unstable,
+and delays in pub/sub.
+
+**Default behavior** - If the property is not specified, the default value is
```

Comment:
how about "is not fefined in the scenario file,"

Reply:
```suggestion
**Default behavior** - If the property is not defined, the default value is
```

Reply:
thanks